src/Contracts/Models/TaskExecution.cs

```diff
@@ -0,0 +1,36 @@
+﻿// SPDX-FileCopyrightText: © 2021-2022 MONAI Consortium
+// SPDX-License-Identifier: Apache License 2.0
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Extensions.Configuration;
+
+namespace Monai.Deploy.WorkflowManager.Contracts.Models
+{
+    public class TaskExecution
+    {
+        [ConfigurationKeyName("execution_id")]
```

Comment:
May want to use `JsonProperty` for models instead of `ConfigurationKeyName`

Reply:
Changed